gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/types.ts

```diff
@@ -0,0 +1,81 @@
+export type OffchainConfig = {
+  deltaProgressNanoseconds: number
+  deltaResendNanoseconds: number
+  deltaRoundNanoseconds: number
+  deltaGraceNanoseconds: number
+  deltaStageNanoseconds: number
+  rMax: number
+  s: number[]
+  offchainPublicKeys: string[]
+  peerIds: string[]
+  reportingPluginConfig: {
+    alphaReportInfinite: boolean
+    alphaReportPpb: number
+    alphaAcceptInfinite: boolean
+    alphaAcceptPpb: number
+    deltaCNanoseconds: number
+  }
+  maxDurationQueryNanoseconds: number
+  maxDurationObservationNanoseconds: number
+  maxDurationReportNanoseconds: number
+  maxDurationShouldAcceptFinalizedReportNanoseconds: number
+  maxDurationShouldTransmitAcceptedReportNanoseconds: number
+  configPublicKeys: string[]
+}
+
+export type InspectionInput = {
```

Comment:
I imagine this change comes from the `NewTransmission` type, but this is not a common type, is only useful in its own context (the command) and I don't think should be in a common place. Also, if we bring every command input type here, the list can be unreadable. 
Let's have the Input types closer to where they are used for now. Same for `OffchainConfig`

Reply:
Going to remove the types file for now until we decide on how we want to restructure going forward. I'll stick with defining objects where they are used for now.